### DISCUSSION TITLE
Crashing browser on pull request list.

### DISCUSSION THREAD

**Discussion by skinkie** [community/community#13875] [Pull Requests]
When I browse to https://github.com/NeTEx-CEN/NeTEx/pulls under a certain zoom level my Chromium browser starts to very fastly flicker as if it would break two lines, then when the lines are broken it is not overflowing anymore. I hope that a literal photo of my screen can clarify the issue better.

![image](https://user-images.githubusercontent.com/502394/160830043-69a5603d-0236-498d-9750-8523bb7a8a72.png)

**Comment by giampaolo** [comment]
Same for me (Brave browser) when I **hover** over the list of PRs.
It's severe because I have to kill the whole browser.
